src/app/components/JumpTo/index.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import JumpTo from '.';
+import { StoryArgs } from '../../models/types/storybook';
+import metadata from './metadata.json';
+import readme from './README.md';
+
+interface Props {
+  subheadlines?: { heading: string }[];
+}
```

Comment:
Can this type be shared in a common place, perhaps exported from the JumpTo component?